pycookiecheat/pycookiecheat.py

```diff
@@ -64,23 +64,30 @@ def chrome_decrypt(encrypted_value: bytes, key: bytes, init_vector: bytes) \
     return clean(decrypted)
 
 
-def get_osx_config() -> dict:
-    """Get settings for getting Chrome cookies on OSX.
+def get_osx_config(browser_name: str = None) -> dict:
+    """Get settings for getting Chrome/Chromium cookies on OSX.
 
     Returns:
-        Config dictionary for Chrome cookie decryption
+        Config dictionary for Chrome/Chromium cookie decryption
 
     """
+    if browser_name is None:
+        browser_name = 'Chrome'
+    if browser_name == 'Chrome':
+        cookie_dir = 'Google/' + browser_name
+    else:
+        cookie_dir = browser_name
     config = {
-        'my_pass': keyring.get_password('Chrome Safe Storage', 'Chrome'),
+        'my_pass': keyring.get_password(
+            browser_name + ' Safe Storage', browser_name),
         'iterations': 1003,
-        'cookie_file': ('~/Library/Application Support/Google/Chrome/Default/'
-                        'Cookies'),
+        'cookie_file': ('~/Library/Application Support/%s/Default/'
+                        'Cookies') % (cookie_dir,),
```

Comment:
Done. I changed some of the other `browser + STRLIT` to `localvar = 'STRLIT'.format()` as well.